QGL/BasicSequences/RB.py

```diff
@@ -9,41 +9,47 @@
 import numpy as np
 from functools import reduce
 
+
 def create_RB_seqs(numQubits, lengths, repeats=32, interleaveGate=None):
-	"""
+    """
 	Create a list of lists of Clifford gates to implement RB.
 	"""
-	if numQubits == 1:
-		cliffGroupSize = 24
-	elif numQubits == 2:
-		cliffGroupSize = 11520
-	else:
-		raise Exception("Can only handle one or two qubits.")
-
-	#Create lists of of random integers
-	#Subtract one from length for recovery gate
-	seqs = []
-	for length in lengths:
-		seqs += np.random.random_integers(0, cliffGroupSize-1, (repeats, length-1)).tolist()
-
-	#Possibly inject the interleaved gate
-	if interleaveGate:
-		newSeqs = []
-		for seq in seqs:
-			newSeqs.append(np.vstack((np.array(seq, dtype=np.int), interleaveGate*np.ones(len(seq), dtype=np.int))).flatten(order='F').tolist())
-		seqs = newSeqs
-	#Calculate the recovery gate
-	for seq in seqs:
-		if len(seq) == 1:
-			mat = clifford_mat(seq[0], numQubits)
-		else:
-			mat = reduce(lambda x,y: np.dot(y,x), [clifford_mat(c, numQubits) for c in seq])
-		seq.append(inverse_clifford(mat))
-
-	return seqs
+    if numQubits == 1:
+        cliffGroupSize = 24
+    elif numQubits == 2:
+        cliffGroupSize = 11520
+    else:
+        raise Exception("Can only handle one or two qubits.")
+
+    #Create lists of of random integers
+    #Subtract one from length for recovery gate
+    seqs = []
+    for length in lengths:
+        seqs += np.random.random_integers(0, cliffGroupSize - 1,
+                                          (repeats, length - 1)).tolist()
+
+    #Possibly inject the interleaved gate
+    if interleaveGate:
+        newSeqs = []
+        for seq in seqs:
+            newSeqs.append(np.vstack((np.array(
+                seq, dtype=np.int), interleaveGate * np.ones(
+                    len(seq), dtype=np.int))).flatten(order='F').tolist())
```

Comment:
This is one case where the algorithm seems to fail to find a better indentation. Mark it with a `# yapf: disable`?